src/main/java/com/socialmedia/controller/FileHandlerController.java

```diff
@@ -0,0 +1,28 @@
+package com.socialmedia.controller;
+
+import com.socialmedia.service.AmazonClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/storage/")
+public class FileHandlerController {
+
+    private AmazonClient amazonClient;
+
+    @Autowired
+    FileHandlerController(AmazonClient amazonClient) {
+        this.amazonClient = amazonClient;
+    }
+
+    @PostMapping("/uploadFile")
+    public String uploadFile(@RequestPart(value = "file") MultipartFile file) {
+        return this.amazonClient.uploadFile(file);
+    }
+
+    @DeleteMapping("/deleteFile")
```

Comment:
using camelCase is not accepted for urls, it would be better to use simple '/delete'